src/core/border_router/routing_manager.cpp

```diff
@@ -362,7 +362,11 @@ void RoutingManager::Stop(void)
 
     SendRouterAdvertisement(kInvalidateAllPrevPrefixes);
 
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_USE_HEAP_ENABLE
```

Comment:
I see this defaults to `1`. Do we have a build check for `0`?

Reply:
I don't think we have a build covering it. I agree would be good to add and cover it. Can add it in this PR itself

Reply:
Updated the PR.